src/packs/cli.rs

```diff
@@ -132,6 +136,11 @@ pub fn run() -> Result<(), Box<dyn std::error::Error>> {
             Ok(())
         }
         Command::Check { files } => checker::check_all(configuration, files),
+        Command::CheckContents { file } => {
+            let absolute_path = get_absolute_path(file.clone(), &configuration);
```

Comment:
One option is to check if the input file path is a file and that it exists and erroring if not. This could always be added as a follow-up, especially since it's not expected the client (i.e. the ruby mine extension) passes in directories or non-existent files.